service/src/main/resources/api/service_openapi.yaml

```diff
@@ -277,6 +302,22 @@ components:
       description: >
         The total number of billing profiles available and a page of profiles
 
+    AzureManagedAppsResponseModel:
+      type: object
+      properties:
+        managedApps:
+          type: array
+          items:
+            $ref: '#/components/schemas/AzureManagedAppModel'
+
+    AzureManagedAppModel:
```

Comment:
Wondering if (tenant, subscription, name) should be in the response, since presumably the caller will use this info to create a billing profile?